File: Unit 1/IncreasingOrder/InOrder.py
def order(a:int, b:int, c:int):
    if a<b and a<c:
        if b<c:
            print(a, b, c)
        else:
            print(a, c, b)
    
    elif (b<a and b<c):
        if a<b:
            print(b, a, c)
        else:
            print(b, c, a)
    
    else:
        if a<b:
            print(c, a, b)
        else:
            print(c, b, a)

order(7,3,6)

File: Unit 1-Into/IncreasingOrder/InOrder.cpp
#include <iostream>

using namespace std;

void order(int a, int b, int c)
{
    if (a < b && a < c)
    {
        if (b < c)
        {
            cout << a << " " << b << " " << c << endl;
        } else
        {
            cout << a << " " << c << " " << b << endl;
        }
    } else if (b < a && b < c)
    {
        if (a < c)
        {
            cout << b << " " << a << " " << c << endl;
        } else {
            cout << b << " " << c << " " << a << endl;
        }
    } else
{
        if (a < b)
        {
            cout << c << " " << a << " " << b << endl;
        } else {
            cout << c << " " << b << " " << a << endl;
        }
    }
}

int main()
{
    order(7, 3, 6);
    return 0;
}
